### DISCUSSION TITLE
CKEditor and mj-buttons

### DISCUSSION THREAD

**Discussion by pepoteloko** [GrapesJS/grapesjs#6175] [Q&A]
Hi, we are using MJML + CKEditor, but our clients always add links inside mj-buttons 🥇 

I'm not really sure if this is normal or it's something we activate, but there is a way to disable CKEditor inside mj-buttons?

**Comment by RakulAgn** [comment]
@pepoteloko Hey I hope this is what your asking you can prevent showing ckeditor by finding the component type if its mj-button you can just prevent to show the ckeditor



https://github.com/user-attachments/assets/048a0fc7-ba15-4ded-a778-afd945b0f0f5

**Reply by pepoteloko** [reply]
Yes, this is what I want to achieve. CKEditor only in non-button elements.
